new-frontend/src/app/components/permissions-matrix/permissions-matrix.component.ts

```diff
@@ -102,7 +115,7 @@ export class PermissionsMatrixComponent implements OnInit, OnChanges, OnDestroy
   }
 
   ngOnChanges(changes: SimpleChanges): void {
-    if (changes['fixedSelected'] || changes['preselected'] || changes['selectable']) {
+    if (changes['fixedSelected1'] || changes['fixedSelected1'] || changes['preselected'] || changes['selectable']) {
```

Comment:
I *think* the second one should be `fixedSelected2`?

Reply:
I changed this already in a commit where I'm renaming the variable, but would love if you could check :-)

Reply:
I saw you also renamed this in another commit, so let's merge it!